.github/actions/move-released-issues-and-create-sync-pr-v1/action.yml

```diff
@@ -0,0 +1,52 @@
+name: move-released-issues-and-create-sync-pr-v1
+description: Label all related issues with the package version and create a PR to sync branches
+
+inputs:
+  token:
+    description: 'A GitHub token with the required permissions'
+    required: true
+  project-number:
+    description: 'A project number of the project board'
+    required: false
+    default: '66'
+  head:
+    description: 'A head branch to sync from'
+    required: false
+    default: main
+  base:
+    description: 'A target branch for the created pull request'
+    required: false
+    default: develop
+  pr-team-reviewers:
+    description: 'Reviewers to tag on the created pull request'
+    required: false
+    default: axe-api-team
+
+runs:
+  using: 'composite'
+  steps:
+    # This step is required for the next steps
+    - uses: actions/checkout@v4
+      with:
+        # Fetch all history
+        fetch-depth: 0
+    - uses: dequelabs/axe-api-team-public/.github/actions/get-package-version-v1@main
+      id: get-package-version
+    - uses: dequelabs/axe-api-team-public/.github/actions/generate-commit-list-v1@main
+      id: get-commit-list
+      with:
+        tag: v${{ steps.get-package-version.outputs.version }}
```

Comment:
This will need to be the tag prior to the current one. If v4.9.0 is released we need the tag to be the prior one (e.g. v4.8.4) in order to get the list of commits from the last release to the current HEAD. Otherwise the current tag on main/master is the current HEAD